x/poolmanager/keeper.go

```diff
@@ -34,6 +34,9 @@ type Keeper struct {
 	poolModules []types.PoolModuleI
 
 	paramSpace paramtypes.Subspace
+
+	defaultTakerFeeBz  []byte
```

Comment:
We need to initialize this value somewhere right?

Reply:
no its intended to be set to nil until initialization!